packages/system-tests/assets/sfdx-simple/force-app/main/default/classes/TestApexTrigger.trigger

```diff
@@ -0,0 +1,3 @@
+trigger TestApexTrigger on SOBJECT (before insert) {
```

Comment:
Is this also needed for this PR?

Reply:
No. These are leftover test artifacts. I've removed them now.